english/necklace/necklace.md

```diff
@@ -3,4 +3,4 @@ Create 5 sets of 5 marshmallows each for a total of 25 in all,
 Stick 4 marshmallows together from each set together like a string,
 Separate each set with another marshmallow half an inch in between,
 For sure you will enjoy the look of each bead,
-Marshmallow here I come I wil eat you in peace.
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.